Ports/doxygen/package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=doxygen
+version='1.9.7'
+files=(
+    "https://github.com/doxygen/doxygen/archive/refs/tags/Release_${version//./_}.tar.gz 691777992a7240ed1f822a5c2ff2c4273b57c1cf9fc143553d87f91a0c5970ee"
+)
+workdir="${port}-Release_${version//./_}"
+useconfigure='true'
+configopts=(
```

Comment:
This should presumably include `-DCMAKE_TOOLCHAIN_FILE=...`, which might also remove the need for some of the manual settings here.